dask/bag/core.py

```diff
@@ -146,6 +146,14 @@ def to_textfiles(b, path, name_function=str, compression='infer',
 
     **Compression**: Filenames with extensions corresponding to known
     compression algorithms (gz, bz2) will be compressed accordingly.
+    
+     **Bag Contents**: The bag calling ``to_textfiles`` _must_ be a bag of
+    text strings. For example, a bag of
+    dictionaries could be written to JSON text files by mapping ``json.dumps``
+    on to the bag first, and then calling ``to_textfiles``:
+
+    >>> b_dict.map(json.dumps).to_textfiles("/path/to/data/*.json")
```

Comment:
This looks good to me.  I suggest wrapping text so that each line starts on the same column (`**Bag Contents**` is one space indented) and that we fill out to 80 character lines (dictionaries should be on the line above).